components/update-overview/update-overview.less

```diff
@@ -4,53 +4,76 @@
 .update-overview {
   .clearfix();
 
-  .update-extract-small {
-    max-width: 450px;
-    margin-right: auto;
-    margin-left: auto;
+  > .container-inner {
```

Comment:
this looks weird to me. not sure if this works or is needed. isnt it the save if you would remove the >

Reply:
We need this styling only on the immediate child `.container-inner`, not on any other `.container-inner`.

Reply:
Ah yes I understand